modules/module-extras.php

```diff
@@ -49,6 +49,11 @@
 	'simple-payments/simple-payments.php',
 	'wpcom-block-editor/class-jetpack-wpcom-block-editor.php',
 	'wpcom-tos/wpcom-tos.php',
+	// These oEmbed providers are available when connected to WordPress.com.
+	// Starting from 2020-10-24, they need an authentication token, and that token is stored on WordPress.com.
+	// More information: https://developers.facebook.com/docs/instagram/oembed/.
+	'shortcodes/facebook.php',
```

Comment:
Does the Facebook embed require a connection to WordPress.com? If not, it could be moved to the array above?
https://github.com/Automattic/jetpack/blob/b62a336b19ebeb2d1590753f8c4167acab0870b8/modules/shortcodes/facebook.php

Reply:
It does not, since we use the Facebook SDK which does not require auth in the first place.

Reply:
Right, the FB embed is indeed using the "JS SDK" ([[1]](https://developers.facebook.com/docs/plugins/embedded-posts/#add-code-manually),[[2]](https://developers.facebook.com/docs/javascript/quickstart)):

https://github.com/Automattic/jetpack/blob/f497482004e2ef00770c8a01f6cff93029be1ef3/modules/shortcodes/facebook.php#L88
https://github.com/Automattic/jetpack/blob/d8fe42cde874e5ef2019ca4ad33491b9a3045e92/class.jetpack.php#L1196-L1202
https://github.com/Automattic/jetpack/blob/d8fe42cde874e5ef2019ca4ad33491b9a3045e92/_inc/facebook-embed.js#L13

It seems like that API is totally distinct and thus unaffected by FB's retiring of their [legacy oEmbed API](https://developers.facebook.com/docs/plugins/oembed-legacy/), so it will continue to work after Oct 24, without the need for an authentication token. This means we can move this line into the `$tools` array :+1: 

(I'm considering dropping the FB embed handler in favor of the [new oEmbed API](https://developers.facebook.com/docs/plugins/oembed) as part of the embeds simplification project that I'm hoping to kick off soon, in which case we'll have to move this line back into the `$connected_tools` array, but we can cross that river when we reach it.)

Reply:
16dbe6e